CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+### v3.0
+
+- Changed autoload from PSR-0 to PSR-4
+- v3 now supports Symfony 4.3+ and Symfony 5.0+
+- Uses version ~2.8 of `mixpanel/mixpanel-php`
```

Comment:
Can you add all the changes in your french comment here? But in English, please :)